sqe-http-api/Services/UserService.cs

```diff
@@ -19,28 +19,40 @@ public interface IUserService
         UserDTO GetCurrentUser();
         uint? GetCurrentUserId();
         UserInfo GetCurrentUserObject(uint? editionId = null);
+        Task<UserDTO> CreateNewUserAsync(NewUserDTO newUserData);
```

Comment:
Ooooh. It's the `NewUserRequestDTO` ! You should rename the class, then.

Reply:
Thanks.  It is renamed now.

Done.